aana_chat_with_video/endpoints/index_video.py

```diff
@@ -162,7 +162,7 @@ async def run(  # noqa: C901
 
                 timestamps.extend(frames_dict["timestamps"])
                 frame_ids.extend(frames_dict["frame_ids"])
-                chat_prompt = "Describe the content of the following image in a single sentence:"
+                chat_prompt = "Caption the content of the following image in a single and concise sentence"
```

Comment:
@HRashidi Can you try to call the captioning model not in batches but by scheduling multiple asyncio tasks like here https://github.com/mobiusml/aana_episodic_retrieval/blob/1a431ed39f70c01af72ccbbb89aeb66271ce38af/aana_episodic_retrieval/endpoints/episodic_summary.py#L120? It should be much faster.